### ISSUE TITLE
[feature]:  Add "**Does not include**" filter capability

### ISSUE THREAD

**Issue by marcungeschikts** [makeplane/plane#3316]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Be able to create a view that does not include a specific value of property like at least for labels

### Why should this be worked on?

When there are a lot of values for a property like often for labels, it is not user friendly to select all the fields except one to  not include this specific label.

**Comment by mmospanenko** [comment]
yes, crucially functionality. Not hard to implement but very important.

Filter should be able to use logic: and, or, contains, exact, etc. Like Notion has (groups of filters)

For example, I want to see all issues without Cycles
